### DISCUSSION TITLE
Adding drivers R&S ZNLE series of VNAs

### DISCUSSION THREAD

**Discussion by FKMalina** [microsoft/Qcodes#3975] [Ideas]
It is a series of cheap VNAs from R&S with a lot of functionality of the ZNB series, in particular, easily sufficient for application in reflectometry.
ZNLE is a very similar instrument to ZNB, and huge portions of the ZNB drivers could be repurposed, or even ZNB driver could modified to allow for use with ZNLE series.

**Comment by jenshnielsen** [comment]
@FKMalina thank you for your question. If someone is willing to write a driver for that vna reusing parts of the existing driver we are happy to help with review etc and make it part of qcodes.
